fastlane/Fastfile

```diff
@@ -541,6 +541,58 @@ lane :beta do
   )
 end
 
+lane :download_app_store_reviews do
+  desc "Download App Store reviews and save them to a file"
+
+  UI.message "Logging into App Store Connect..."
+
+  # Use credentials from the Appfile
+  apple_id = CredentialsManager::AppfileConfig.try_fetch_value( :apple_id )
+  itc_team_id = CredentialsManager::AppfileConfig.try_fetch_value( :itc_team_id )
+
+  # Explicitly login with credentials
+  Spaceship::Tunes.login( apple_id )
+  Spaceship::Tunes.select_team( team_id: itc_team_id, team_name: nil )
+
+  # Select the app using the bundle ID from the Appfile
+  UI.message "Finding app with bundle ID: #{IOS_BUNDLE_ID}"
+  app = Spaceship::Tunes::Application.find( IOS_BUNDLE_ID )
+
+  unless app
+    UI.abort_with_message! "Could not find app with bundle ID: #{IOS_BUNDLE_ID}"
+  end
+
+  # Fetch reviews
+  UI.message "Fetching reviews for #{app.name}..."
+  reviews = app.ratings.reviews
```

Comment:
are reviews paginated? do we need to consider downloading multiple pages of reviews, or are you only trying to fetch the most recent page of reviews? i'm not sure what this code will ultimately be used for or where this information is going... are you just collecting this locally or planning to share with the broader team?

Reply:
The purpose is to share with the broader team, especially for review analyses.
I have tested the same script in Seek now with all reviews and it did not need any pagination.